src/maintenance/packages/non-free.md

```diff
@@ -0,0 +1,37 @@
+# Non-free packages
+
+Void has a `nonfree` repository for packages that don't have free licenses.
+Install the `void-repo-nonfree` package to access this repository:
+
+```
+# xbps-install -S void-repo-nonfree
+```
+
+If you are using a different mirror than the default, change the mirror in the
+configuration file:
+
+```
+/etc/xbps.d/10-repository-nonfree.conf
+```
+
+## Restricted packages
+
+There are some packages that are supported but do not allow redistribution of
```

Comment:
`but their licenses do not allow`

Reply:
And `A step beyond non-free, there are some packages`

Reply:
Good call